appuio/k8up/Chart.yaml

```
@@ -6,11 +6,10 @@ keywords:
   - backup
   - operator
   - restic
-version: 1.1.0
-appVersion: v1.2.0
+version: 1.2.0
+appVersion: v2.0.0
```

I think due to the breaking changes in K8up usage, we should also bump this to v2 as well, so that users do not blindly upgrade seemingly minor version
```suggestion
version: 2.0.0
appVersion: v2.0.0
```